libraries/resource_install.rb

```diff
@@ -0,0 +1,22 @@
+class Chef
+  # Chef Resource for installing or removing Elasticsearch from package or source
```

Comment:
As I said above, examples would usually be resource examples, and would go in the README, since users don't usually instantiate these classes directly.